.github/workflows/build.yml

```
@@ -41,48 +41,69 @@ jobs:
 
+      - name: Set up Docker Compose
+        run: docker compose -f docker-compose.yml up -d
+
+      - name: Wait for SQL Server to be ready
+        run: |
+          echo "Waiting for SQL Server to start..."
+          sleep 20
+
       - name: Run unit tests and generate coverage
-        run: dotnet-coverage collect "dotnet test VictoryCenter/VictoryCenter.UnitTests/VictoryCenter.UnitTests.csproj --configuration Release" -f xml -o "coverage.xml"
+        run:  dotnet test VictoryCenter/VictoryCenter.sln /p:CollectCoverage=true /p:CoverletOutputFormat=opencover /p:CoverletOutput=../coverage.xml /p:Exclude="[*.Tests]*"
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)